### ISSUE TITLE
[Feature Request] Use the API to get name of products

### ISSUE THREAD

**Issue by Dtrieb** [lorenzovngl/FoodExpirationDates#133]
Use from this site the free API to get from a scanned barcode of product the name and brand.

This would fill more easier than typing.

https://world.openfoodfacts.org/

Example:

https://de.openfoodfacts.org/api/v2/product/3017620425035?fields=brands,product_name,code

**Comment by Dtrieb** [comment]
Example:

https://de.openfoodfacts.org/api/v2/product/3017620425035?fields=brands,product_name,code

**Comment by lorenzovngl** [comment]
Hi!
Thanks for the suggestion, this has been on my to-do list for a while and will be the next feature I will implement.

**Comment by lorenzovngl** [comment]
Hello,

I've started working on the product retrieval feature using the barcode scanner. I've pushed an initial implementation to the branch: [133-feature-request-use-the-api-to-get-name-of-products](https://github.com/lorenzovngl/FoodExpirationDates/tree/133-feature-request-use-the-api-to-get-name-of-products). Although it still needs some improvements, this provides a first look at what the feature will look like.

I welcome any feedback you might have.
